Claim redirect_uri on token request (#209)
italia/spid-cie-oidc-docs: docs/en/token_endpoint.rst

    client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwtbearer&
    code=usDwMnEzJPpG5oaV8x3j&
    code_verifier=9g8S40MozM3NSqjHnhi7OnsE38jklFv2&
    grant_type=authorization_code&

>     redirect_uri=https://rp.spid.agid.gov.it/redirect

Reviewer: ```suggestion
    redirect_uri=https%3A%2F%2Frp.spid.agid.gov.it%2Fredirect
```

    ibmFtZSI6IlNQSUQiLCJhZG1pbiI6dHJ1ZX0.LVyRDPVJm0S9q7oiXcYVIIqGWY0wWQlqxvFGYswL…&
    client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwtbearer&
    grant_type=refresh_token&
    refresh_token=8xLOxBtZp8&

>     redirect_uri=https://rp.spid.agid.gov.it/redirect

Reviewer: ```suggestion
    redirect_uri=https%3A%2F%2Frp.spid.agid.gov.it%2Fredirect
```

   * - **refresh_token**
     - Required only if **grant_type** is **refresh_token**.
     - |spid-icon| |cieid-icon|
   * - **redirect_uri**

>      - Required. It must be one of the values declared into the claim **redirect_uris** in the metadata of the RP and it must be identical to the value of the claim **redirect_uri** included in the authorization request.

Reviewer: ```suggestion
     - Required. It must be one of the values declared into the claim **redirect_uris** in the metadata of the RP and it MUST be identical to the value of the claim **redirect_uri** included in the authorization request.
```

   * - **refresh_token**
     - Required only if **grant_type** is **refresh_token**.
     - |spid-icon| |cieid-icon|
   * - **redirect_uri**

>      - Required. It must be one of the values declared into the claim **redirect_uris** in the metadata of the RP and it must be identical to the value of the claim **redirect_uri** included in the authorization request.

Reviewer: ```suggestion
     - Required. It MUST be one of the values declared into the claim **redirect_uris** in the RP metadata and it MUST match the value of the claim **redirect_uri** included in the previous authorization request.
```

>      - |spid-icon| |cieid-icon|

Reviewer 2: I don't see any normative section within OIDC Core and iGov Profile requiring redirect_uri as a mandatory claim in the token request. Moreover, I see no security issues if this claim is omitted. I suggest considering it as OPTIONAL saying that if it is present the OP MUST check that it is the same value provided in the authorization request and the same value included in the RP Metadata.
